shell/browser/api/electron_api_session.cc

```diff
@@ -307,7 +307,6 @@ Session::~Session() {
   // Refs https://github.com/electron/electron/pull/12305.
   DestroyGlobalHandle(isolate(), cookies_);
   DestroyGlobalHandle(isolate(), protocol_);
-  DestroyGlobalHandle(isolate(), net_log_);
```

Comment:
NB. this is no longer needed because the object will be GC'd by V8.

Reply:
Some session properties hold `scoped_refptr` to `ElectronBrowserContext` while some hold a weak-reference, we should ensure `ElectronBrowserContext` is released when session is destroyed and no other reference exits, otherwise bad things can happen on shutdown.